codevalidator-api/src/main/java/org/sitenv/vocabularies/servlet/listener/VocabularyValidationListener.java

```diff
@@ -144,56 +135,22 @@ public void contextInitialized(ServletContextEvent arg0) {
 		}
 	}
 	
-	private VocabularyRepositoryConnectionInfo loadConnectionInfo(String configFile, String dbConnectionName) 
-	{
-		try {
-			
-			DocumentBuilderFactory domFactory = DocumentBuilderFactory.newInstance();
-			DocumentBuilder builder = domFactory.newDocumentBuilder();
-			
-			Document doc  = builder.parse(configFile);
-			
-			if (doc != null)
-			{
-				XPath xpath = XPATH.newXPath();
-				XPathExpression expExpressionNode = xpath.compile("/orient-server/storages/storage[@name='"+ dbConnectionName +"']");
-				XPathExpression expUserPasswordAtt = xpath.compile("@userPassword");
-				XPathExpression expUserNameAtt = xpath.compile("@userName");
-				
-				Node expressionNode = (Node)expExpressionNode.evaluate(doc, XPathConstants.NODE);
-				
-				String userName = (String)expUserNameAtt.evaluate(expressionNode, XPathConstants.STRING);
-				String userPassword = (String)expUserPasswordAtt.evaluate(expressionNode, XPathConstants.STRING);
-				
-				VocabularyRepositoryConnectionInfo config = new VocabularyRepositoryConnectionInfo ();
+	private VocabularyRepositoryConnectionInfo loadConnectionInfo(OServerConfiguration serverConfiguration, String dbConnectionName) {
```

Comment:
OrientDB parses it's configuration into beans anyway - no need to manually extract parts.